packages/core/src/components/TabView/TabView.tsx

```diff
@@ -111,7 +111,7 @@ const TabViewComponent: React.FC<React.PropsWithChildren<TabViewProps>> = ({
         indicatorStyle={{
           backgroundColor: indicatorColor || theme.colors.primary,
         }}
-        labelStyle={textStyles}
+        labelStyle={[textStyles, { textTransform: "none" }]}
```

Comment:
```suggestion
        labelStyle={[{ textTransform: "none" }, textStyles]}
```
I'd reverse the order, so the user is still able to override it if needed